backend/lucene/src/main/java/org/hibernate/search/backend/lucene/types/dsl/impl/StringIndexSchemaFieldContext.java

```diff
@@ -73,10 +73,9 @@ public StringIndexSchemaFieldContext sortable(Sortable sortable) {
 	@Override
 	protected void contribute(IndexSchemaFieldDefinitionHelper<String> helper, LuceneIndexSchemaNodeCollector collector,
 			LuceneIndexSchemaObjectNode parentNode) {
-		// TODO For now, we allow the sortable - analyzed combination, it will be disallowed later
-//		if ( Sortable.YES.equals( getSortable() ) && analyzer != null ) {
-//			throw log.cannotUseAnalyzerOnSortableField( getRelativeFieldName() );
-//		}
+		if ( Sortable.YES.equals( sortable ) && analyzer != null ) {
+			throw log.cannotUseAnalyzerOnSortableField( getSchemaContext().getEventContext() );
+		}
```

Comment:
In the Es backend, you have this `switch` construct. Either it brings something to the plate (having a warning when a new value is added) and we should use it in both backends or it doesn't and I think we should simplify the Es backend.